src/components/Card.jsx

```diff
@@ -1,50 +1,50 @@
-import React from "react";
-import jsonData from "../DB/exerciseData.json";
+import React from 'react'
+import jsonData from '../DB/exerciseData.json'
 
 export default function Card() {
   return (
-    <div className="grid grid-cols-1 lg:grid-cols-2 p-3">
+    <div className='grid grid-cols-1 lg:grid-cols-2 p-3'>
       {jsonData.map((exercise, index) => (
         <div
           key={index}
-          className=" max-w-md mx-auto my-4 bg-white rounded-xl shadow-md overflow-hidden md:max-w-2xl hover:border-purple-500 border-2"
+          className='mx-3 max-w-md mx-auto my-4 bg-white rounded-xl shadow-md overflow-hidden md:max-w-2xl hover:border-purple-500 border-2'
```

Comment:
If `mx-auto` is present alongside `mx-3`, the `mx-3` won't work. To maintain consistency with our app's code style, we use double quotation marks instead of single ones. So would you please revert the changes you made to switch from double to single quotation marks?